src/functions.ts

```diff
@@ -36,8 +41,16 @@ function isOdd(n: number): boolean {
  */
 function oddsSmallerThan(n: number): number {
   // Your code here
-
-  return -1; // replace -1 with what you see is fit
+  var counter = 0;
+  if(n === 1) return 0;
+  n = n-1;
+  while (n > 0) {
+    if(n%2 !== 0){
+      counter++;
+    }
+    n--;
+  }
+  return counter; // replace -1 with what you see is fit
```

Comment:
This could have been solved without a loop ;)

```ts
function oddsSmallerThan(n: number): number {
  if (isOdd(n)) return (n + 1) / 2;
  return n / 2;
}
```
OR
```ts
function oddsSmallerThan(n: number): number {
  return (n + (isOdd(n) ? 1 : 0)) / 2;
}
```